docs/content/docs/6.deploy/2.serverless.md

```diff
@@ -99,6 +99,20 @@ The generated output will be compatible with the selected platform.
 The linked database will be loaded on the server side when booting the server. In the browser, a [WASM SQLite](/docs/advanced/database#wasm-sqlite-in-browser) database will be loaded for client-side navigation and actions.
 ::
 
+::tip
+If you wish to deploy to AWS Lambda, you need to make sure your sqlite file is in `/tmp` since this is the only writeable folder
+
+```ts
+export default defineNuxtConfig({
+  modules: ['@nuxt/content'],
+  content: {
+    database: {
+      type: 'sqlite',
+      filename: '/tmp/contents.sqlite',
+    },
```

Comment:
```suggestion
export default defineNuxtConfig({
  modules: ['@nuxt/content'],
  content: {
    database: {
      type: 'sqlite',
      filename: '/tmp/contents.sqlite'
    }
})
```

Reply:
`database` is only for production, module uses a separate internal config (`_localDatabase`) for local/dev.